src/psij/web/js/RowAccordian.js

```diff
@@ -0,0 +1,66 @@
+var VC = VC || {};
+
+VC.goHere = function(ev) {
+
+    var x = $(event.target);
+    var site_id = x.html();
+
+    window.location="site.html?site_id=" + site_id;
+};
+
+
+VC.RowAccordian = Vue.component("row-accordian", {
```

Comment:
the accordian we need, needs to have rows and columns that span across the body of the accordian to the clickable row of the accordian.  Also, we need the arrow of the accordian to change direction when you click on it.

Reply:
Sorry, but I disagree, just as I did before. There are expansion panels (https://vuetifyjs.com/en/components/expansion-panels/#usage) in vuetify which can be used instead. The icons can be customized, so that is not an issue.

The content of the expanders need not be table rows, but I do not see why they couldn't be.

However, even if so, vuetify has tables where rows can be grouped or expanded: 
https://vuetifyjs.com/en/components/data-tables/#grouping 
https://vuetifyjs.com/en/components/data-tables/#expandable-rows

Reply:
I will resolve this for now. For reference, I do not think the reasoning behind having a custom component applies much since the current solution (with the custom components) is not to expand rows in a table, but to have multiple tables that happen to align perfectly thus giving the appearance of a single table. But this takes extra effort to ensure that the columns form multiple tables are aligned and could as well be done better with plain divs.